### ISSUE TITLE
core-common.js bundle size

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/server#36384]
After #36383 there is still 10 MB left:

![Screenshot 2023-01-26 at 14 11 17](https://user-images.githubusercontent.com/3404133/214843917-269656a4-126e-4449-b62d-75ee1fa21636.png)

- emoji-mart-vue-fast is included twice
- `@nextcloud/password-confirmation` seems quite large
- `@nextcloud/dialogs` loads both index.es.js and index.js

**Comment by clickfreak** [comment]
Every time when I send link to shared file or poll to someone I have following chat:
- link doesn't work
- please wait some time, don't close the page
- strange, loaded but so long...

**Comment by juliushaertl** [comment]
https://github.com/nextcloud/server/pull/43553 will contribute further to that https://github.com/nextcloud/server/pull/43553

**Comment by felixfischer** [comment]
Are there any plans to work on reducing the bundle size?

**Comment by asavageiv** [comment]
Nextcloud is all but unusable on the slow WiFi of the train I'm on largely due to this. It looks like #43553 is no longer making progress. Is help needed to take it over or has it been superseded by other work?
